DEBUGGING.md

```diff
@@ -0,0 +1,62 @@
+# Debugging
+
+## Using the `cpu` command
+
+Since the stboot image running on a server has much fewer tools and services than usual Linux operating systems, the `cpu` command is the best option for debugging a remote machine. It connects to the remote server, bringing all your local tools and environment with you. It's like magic :)
+
+### Prerequisites
+
+These instructions expect _your system_ to be Linux-based.
+You need to have the `go` programming language installed on _your system_.
+
+It also expects you to have access to the serial console of your _remote target_.
+Your _remote target_ needs port `2222` to be accessible from the outside.
+
+### Installation
+
+Run:
+
+```shell
+$ go install github.com/u-root/cpu/cmds/cpu
```

Comment:
In addition we need to get the _cpuserver_ command to include it into the initramfs later (`go get github.com/u-root/cpu/cmds/cpuserver`)

However, since _cpuserver_ not available in _master_ branch but in _configs_, I have had to to the following:
```
go get github.com/u-root/cpu/cmds/cpu
cd $GOPATH/src/github.com/u-root/cpu
git checkout --track origin/configs
```
Do we need to take _cpuserver_ from _configs_ brnach or can we go with _cpud_ from _master_ branch as well?

Reply:
This is all because Ron just renamed the `cpuserver` cmd to `cpud` :disappointed: 
A quick rename should make it work

Reply:
```
go get github.com/u-root/cpu/cmds/cpud
go install github.com/u-root/cpu/cmds/cpud
```
should then do the trick

Reply:
OK so we will go with the master branch and cpud? 
If so, please update the docu, `stboot/make_initramfs.sh` and `stboot/include/start_cpu.elv`